### PR TITLE
GDB-7986: Added feature toggles

### PR DESCRIPTION
**Pull request by mihailradkov**
Toggles include:

- `cluster.jobs.createCluster.enabled`
- `cluster.jobs.patchCluster.enabled`
- `cluster.jobs.scaleCluster.enabled`
- `headlessService.enabled`
- `proxy.service.enabled`
- `proxy.headlessService.enabled`

Additionally

- Removed all useless $.
- Removed the default logback XML configuration and configmap in favor of an [example](examples/custom-logback) and a new configuration options under `logging.logback`
- Configmaps from `graphdb.configs` are now under `extraConfiguration`, `repositories` and `initialConfiguration` with a different structure allowing better reuse of existing configmaps
- Existing configmaps for `logging.logback` and `extraConfiguration.properties` are now templated

### CODE REVIEW

File: templates/graphdb/statefulset.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
     {{- tpl ( toYaml . ) $ | nindent 4 }}
   {{- end }}
 spec:
-  replicas: {{ $.Values.replicas }}
+  replicas: {{ .Values.replicas }}
   serviceName: {{ include "graphdb.fullname.service.headless" . }}
   updateStrategy:
     type: RollingUpdate
```
Comment (yaskoo) [inline templates/graphdb/statefulset.yaml:15]: We can think about making this a config. I think `OnDelete` is better in our case.

File: values.yaml [hunk 2]
```diff
@@ -55,19 +55,23 @@ image:
 ##########################
 
 # Number of GraphDB nodes to be deployed as part of the StatefulSet.
-# Set value to 1 to run a standalone GraphDB instance or to more to form a GraphDB cluster.
+# Set value to 1 to run a standalone GraphDB instance or to more to form a GraphDB cluster with GraphDB cluster proxies.
 replicas: 1
```
Comment (yaskoo) [inline values.yaml:59]: Is this used for the proxy replica count as well?

**Review by yaskoo — APPROVED** [review]

File: templates/graphdb/statefulset.yaml (hunk 1 above)
Comment (mihailradkov) [reply templates/graphdb/statefulset.yaml:15]: Already on my TODO list

File: values.yaml (hunk 2 above)
Comment (mihailradkov) [reply values.yaml:59]: Nope, the proxy has another one `proxy.replicas`

**Pull request merged**